plot_likert/plot_likert.py

```diff
@@ -49,6 +49,8 @@ def plot_counts(
     figsize=None,
     xtick_interval: typing.Optional[int] = None,
     compute_percentages: bool = False,
+    ax: typing.Optional[matplotlib.axes.Axes] = None,
+    **kwargs
```

Comment:
>  is there any reason to provide that argument explicitly

I did it this way because I've seen it as a common practice in other libraries such as pandas and seaborn. I think the reasoning is that the `ax` is not an option of the plot (like, say, `edgecolor` or `width`) but rather the plot itself.

Reply:
Hmm. I can kind of see the reasoning, but, looking at pandas, for example, `ax` isn't an explicit argument [to `plot.barh`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.plot.barh.html), and [in `plot`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.plot.html), it seems that they have `ax` on the same "level" as the other options. Let me know if there are better examples I should be looking at, though.

I think I have a small preference for putting all the "advanced" functionality under `kwargs`, and leaving the library API as simple and easy to understand as possible. (In fact, after your change, we could even get rid of the explicit `figsize` argument!) How would you feel about that?

Reply:
> and [in `plot`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.plot.html), it seems that they have `ax` on the same "level" as the other options

Yeah, you're right, even though it's explicitly documented. I personally also prefer it being explicit because I don't think the ax is an obvious option (at least when I first encountered matplotlib) like the rest. However, I'm okay with moving it to kwargs.

> In fact, after your change, we could even get rid of the explicit figsize argument!

I thought about it. My only concern is that `xtick_interval` is after `figsize` so removing `figsize` will break `plot_likert`'s usage when all parameters are passed by position. I suspect that's rare because there are a lot of params. I think given that the major version is still 0, this type of breaking change is ok but it's your call.

Reply:
> My only concern is that `xtick_interval` is after `figsize` so removing `figsize` will break `plot_likert`'s usage when all parameters are passed by position.

Yeah, and I don't want to break people's code for no good reason, so I'd like to be pretty careful. I would consider doing something like a deprecation warning and removing it eventually, but I'm not sure how that would work, since I don't want to *remove* the argument, just move it to `kwargs`. So I think I'll keep it as-is for now.

> I personally also prefer it being explicit because I don't think the ax is an obvious option (at least when I first encountered matplotlib) like the rest.

I think that's totally fair. I'm erring on the side of caution to keep the API surface small so that changes (like the one from the comment above) are easier to make.

On the bright side, thanks to your taking the time to update the usage guide with that interesting example, hopefully many people will learn about this option that way. 😄 Thanks again for doing that!